### PR TITLE
Update framework info, better json handling

### PR DESCRIPTION
**Pull request by fafhrd91**
Update db bench commits transaction for each entry in update batch.

### CODE REVIEW

File: frameworks/Rust/actix/src/db.rs [hunk 1]
```diff
@@ -57,12 +58,15 @@ impl Handler<UpdateWorld> for DbExecutor {
     fn handle(&mut self, msg: UpdateWorld, _: &mut Self::Context) -> io::Result<()> {
         use schema::World::dsl::*;
 
-        for world in msg.0 {
-            let _ = diesel::update(World)
-                .filter(id.eq(world.id))
-                .set(randomnumber.eq(world.randomnumber))
-                .execute(&self.0);
-        }
+        let _ = self.0.transaction::<_, Error, _>(|| {
```
Comment (msmith-techempower) [inline frameworks/Rust/actix/src/db.rs]: I don't think this is allowed. To me, this reads like all those updates are going to be batched into a single transaction, and that is against the rules.

Comment (fafhrd91) [reply frameworks/Rust/actix/src/db.rs]: as I understand batch updates is acceptable . section 5, point 9

https://frameworkbenchmarks.readthedocs.io/en/latest/Project-Information/Framework-Tests/#database-updates

```
9. Use of batch updates is acceptable but not required. To be clear: batches are not permissible for selecting/reading the rows, but batches are acceptable for writing the updates.
```

I see some code commits once, for example https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Python/flask/app.py#L159

Comment (fafhrd91) [reply frameworks/Rust/actix/src/db.rs]: here is another example https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Python/pyramid/frameworkbenchmarks/views.py#L86

Comment (nbrady-techempower) [reply frameworks/Rust/actix/src/db.rs]: Batch updates are acceptable, not transactions for batch updating. see rule 10 http://frameworkbenchmarks.readthedocs.io/en/latest/Project-Information/Framework-Tests/#database-updates

Comment (fafhrd91) [reply frameworks/Rust/actix/src/db.rs]: that is the reason for my initial implementation :)

but actix results seems very low for "updates" bench, compared to other benchmarks. I see only reason that other frameworks commits once per batch updates. 
I know python pretty well, two examples I linked commit once for sure, should those results be banned then?

Comment (nbrady-techempower) [reply frameworks/Rust/actix/src/db.rs]: There's a difference between batch processing and transactions. The frameworks you linked to are not using transactions, but a single batch update. This is allowed. If you can do that without using transactions, you're allowed to do that.

Comment (fafhrd91) [reply frameworks/Rust/actix/src/db.rs]: ok, I see. let me change pr

Comment (fafhrd91) [reply frameworks/Rust/actix/src/db.rs]: transaction is removed

**Pull request merged**